eng/scripts/Swagger-Prettier.ps1

```diff
@@ -0,0 +1,43 @@
+[CmdletBinding()]
```

Comment:
How about we rename this file to `Validate-OpenApi-Prettier`, to match `Validate-TypeSpec` ? I dislike the `Swagger Prettier` name from unified pipeline. Plus I think it will help to have different name, to avoid confusing people between old and new prettier.

Reply:
I'm open to calling this Validate-OpenApi-Prettier.